Earthfile

```diff
@@ -567,7 +574,7 @@ ci-tests:
     # BUILD +test-s3
 
 nightly-tests:
-    BUILD +test-python
+    BUILD +test-python --all=1
```

Comment:
Shouldn't we add `BUILD +test-python` to `ci-tests` too?

Reply:
I thought we already ran it as part of CI tests.

I forgot that we disabled it.